[docs]: Fix documentation for the Bounded Box Component

bhojrajnarwae opened the pull request:
I have updated the the changes that were mentioned in issue #11057. The following changes are included:

1. The change in format updates
2. The change in content updates

The changes were implemented in two files :-
Next - ToolJet/docs/docs/widgets/bounded-box.md
Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/bounded-box.md

RudraPratik30 commented on docs/docs/widgets/bounded-box.md:
```diff
@@ -3,6 +3,10 @@ id: bounded-box
 title: Bounded Box
 ---
 
+Here's the updated version with the requested changes:
```
Hey @bhojrajnarwae, you don't need to add this to the documentation. Please remove it.

bhojrajnarwae replied:
@RudraPratik30 I have made the required changes as per given instructions. please check it if everything is fine.

RudraPratik30 commented on docs/docs/widgets/bounded-box.md:
```diff
@@ -131,25 +135,36 @@ A Tooltip is often used to specify the extra information when the user hovers th
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
+
+Here is the updated version of the "Devices" section with the requested changes:
```
Same here, you don't need to add this to the documentation. Please remove it.

bhojrajnarwae replied:
@RudraPratik30 I have made the required changes as per given instructions. please check it if everything is fine.

bhojrajnarwae replied:
@RudraPratik30 please check this PR and and let me know if any change to be made or everything is good to go for merging my PR.

RudraPratik30 reviewed (changes requested):
Hey @bhojrajnarwae please make the mentioned changes in both versions

bhojrajnarwae commented:
@RudraPratik30 please check this PR and and let me know if any change to be made or everything is good to go for merging my PR.

RudraPratik30 commented on docs/docs/widgets/bounded-box.md:
```diff
@@ -29,10 +29,10 @@ A **Bounded Box** is an infinitely customizable image annotation component that
 | List of labels | The list of label that will be displayed in the dropdown while selection in the bounded-box. | Labels in array format: `{{['Tree', 'Car', 'Stree light']}}` |
 
-#### Default value
+### Default value
```
hey this must be a h4 only, can you please revert the changes here?

RudraPratik30 commented on docs/docs/widgets/bounded-box.md:
```diff
@@ -131,25 +131,33 @@ A Tooltip is often used to specify the extra information when the user hovers th
 
 </div>
+<br/>
+<hr/>
```
Can you pls use `----` for adding the divider instead of hr tag?

RudraPratik30 commented on docs/docs/widgets/bounded-box.md:
```diff
@@ -131,25 +131,33 @@ A Tooltip is often used to specify the extra information when the user hovers th
-| Visibility | Toggle on or off to control the visibility of the component when the app is loaded. |  **`{{true}}`** or **`{{false}}`**, By default, it's set to `{{true}}` |
-| Disable | Toggle on to disable the component. | **`{{true}}`** or **`{{false}}`**, By default, it's set to `{{false}}` |
-| Box shadow | Sets the add shadow effects around a component's frame. You can specify the horizontal and vertical offsets(through X and Y sliders), blur and spread radius, and color of the shadow. | Values that represent x,y, blur, spread and color. Ex: `9px 11px 5px 5px #00000040` |
+| Visibility | Toggle on or off to control the visibility of the component when the app is loaded. |  **`{{true}}`** or **`{{false}}`**, By default, it's set to **`{{true}}`** |
```
Can you please add full stops here and wherever necessary in the doc?

RudraPratik30 reviewed (changes requested):
Hey @bhojrajnarwae can you please make the mentioned changes in both the versions?

bhojrajnarwae commented:
@RudraPratik30 I have made the mentioned changes as follows:
1. Reverted the the Default value to h4.
2. Replace the hr tag with ---- for adding the divider.
3. Also added full stops  wherever necessary in the doc.

The following changes are done in following files:
Next - ToolJet/docs/docs/widgets/bounded-box.md
Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/bounded-box.md

karan-rathod-316 reviewed (approved):
Looks good to me!
